docs/pages/getting-started/digitalocean.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: Getting started with Teleport on DigitalOcean
+description: How to install Teleport on DigitalOcean?
+videoBanner: voHQlSX_czE
+---
+
+
+<Admonition
+  type="note"
+  title="Note"
+>
+This tutorial will guide you through quickly getting started with Teleport on DigitalOcean with Teleport 1-Click Droplet app. If you are looking for a manual installation, refer to our [Linux installation guide](./linux-server.mdx). 
+
+</Admonition>
+
+## Prerequisites
+- A Fully Qualified Domain Name (FQDN).
+- A two-factor authenticator app (e.g., [Google Authenticator](https://www.google.com/landing/2step/)).
+- An SSH client like OpenSSH.
+
+## Step 1/3. Install Teleport with 1-Click
+
+Head over to the Teleport page on [DigitalOcean Marketplace](https://marketplace.digitalocean.com/apps/teleport) and click the “Create a Droplet” button:
+
+<Figure align="left" bordered caption="Teleport 1-Click droplet page">
+  ![Teleport 1-Click droplet page](../../img/quickstart/digitalocean/1click-droplet-page.png)
+</Figure>
+
+
+Once you click the button, DigitalOcean redirects you to the control panel to configure resources for the Teleport droplet. This step is similar to how you create a regular [droplet in DigitalOcean](https://docs.digitalocean.com/products/droplets/how-to/create/). Teleport is very lightweight, and if you are just trying out Teleport, you can select the $5 droplet. Make sure you select "SSH keys" as the SSH authentication method as it is more secure than a password.
+<Figure align="left" bordered caption="Create a droplet">
+  ![Create a droplet](../../img/quickstart/digitalocean/create-droplet.png)
+</Figure>
+
+It will take a few minutes before our newly created Teleport droplet is ready. Once the droplet is ready, configure your FQDN with the public IP address of the droplet as an IP address for the `A` record of your domain name. 
+For example, refer to the image below; we use the domain name `teleporters.dev`. The resulting domain we are using as an FQDN is `dodemo.teleporters.dev`, pointing to our Teleport droplet's public IP `192.168.200.200`.
```

Comment:
```suggestion
For example, refer to the image below; we use the domain name `example.com`. The resulting domain we are using as an FQDN is `tele.example.com`, pointing to our Teleport droplet's public IP `192.168.200.200`.
```
Updated to be consistent with https://goteleport.com/docs/getting-started/linux-server/#configure-dns